src/main/java/uk/gov/hmcts/darts/dailylist/service/impl/DailyListUpdater.java

```diff
@@ -162,7 +162,7 @@ private LocalTime getTimeFromTimeMarkingNote(final String timeMarkingNote) throw
                 rawTime = timeMarkingNote;
             }
 
-            return LocalTime.parse(rawTime, new DateTimeFormatterBuilder()
+            return LocalTime.parse(rawTime.strip(), new DateTimeFormatterBuilder()
```

Comment:
it might be better to put the change slightly higher in the code, around line 121 maybe?
![image](https://github.com/hmcts/darts-api/assets/131666106/d3dec2a6-f021-4bbe-8e71-a9c57cc6744d)